pfcpiface/messages.go

```diff
@@ -357,9 +377,11 @@ func (pc *PFCPConn) handleSessionModificationRequest(upf *upf, msg message.Messa
 	for _, uFAR := range smreq.UpdateFAR {
 		var f far
 		var err error
-		if err := f.parseFAR(uFAR, localSEID, upf, update); err != nil {
-			return sendError(err)
+		if err = f.parseFAR(uFAR, localSEID, upf, update); err != nil {
+			log.Println("Parse FAR failed")
+			continue
```

Comment:
same comment as above. why progress when there is an error

Reply:
we are just printing the error here.. even if one of the far is proper we still go ahead and write to fast path. if all FAR parse fails then we respond with error.

Reply:
/cc @thakurajayL

Reply:
discussed with Ajay and removed the skipping of failure part.